.circleci/config.yml

```diff
@@ -51,6 +51,18 @@ executors:
       KUBEBUILDER_ARCH: amd64
       GOLANGCI_LINT_VERSION: 1.43.0
       HELM_VERSION: 3.5.4
+  golang_large:
```

Comment:
For most CI jobs, Docker container [size](https://circleci.com/docs/2.0/configuration-reference/#resourceclass) `medium` is enough to complete the task, but for the lint job, the task crashed after 2.5 minutes (with a 3 minute timeout). Increasing the container size to `medium` (2->3 vCPUs and 4->6GB ram) was sufficient to get a ~1 minute CI job.

Reply:
Any reason to not just switch to the larger size for everything, instead of just linting?

Reply:
@ptnapoleon I am not sure how our billing works, but I figured the others do not require more space to run. If you think it's cleaner to upgrade everything, I'm happy to do that because it keeps the code simpler. What do you think?